### PR TITLE
Checkout notice; closes #121

### LINKED ISSUE #121: As an authenticated lender, when I have pending loans, I will be shown a call to action with a link to complete my loan on every page

### PR DESCRIPTION
**Pull request by oorion**
* Add pending_loans helper to application_controller.rb to allow you to determine if there are any pending loans present
* Update indentation in navbar partial
* Shift certain elements in the navbar right and removed pending loans button
* Add call to action with link to complete loans on every page when a user has pending loans
* Add associated tests
* Remove test that asserted a pending loans button on navbar

### CODE REVIEW

File: test/integration/authenticated_lender_test.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
```
Comment (oorion) [inline test/integration/authenticated_lender_test.rb]: This was generated by rails generate.

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -153,6 +153,10 @@ td {
   @include box-sizing;
 }
 
+#complete_your_loan {
```
Comment (oorion) [inline app/assets/stylesheets/custom.scss]: Not sure why hound doesn't like id selectors.  I think this is a good case to use it.  I'll update the id to use hyphens.

File: app/views/layouts/_nav_bar.html.erb [hunk 3]
```diff
@@ -3,28 +3,25 @@
...
   <% end %>
   <%= menu_group do %>
-    <%= menu_item "Home", root_path %>
-    <% if @tenant %>
-      <%= menu_item "Lend", categories_path(tenant: @tenant.organization) %>
-    <% else %>
-      <%= menu_item "Lend", root_path %>
-    <% end %>
-    <%= menu_item "Pending Loans", pending_loan_path %>
+    <%= menu_item "Lend", categories_path %>
```
Comment (EmilyMB) [inline app/views/layouts/_nav_bar.html.erb]: Maybe we want to add an if here to have this disappear if current_user && current_user.admin? (either in this PR or in a new ticket).

File: app/views/layouts/_nav_bar.html.erb [hunk 4]
```diff
@@ -3,28 +3,25 @@
...
     <%= menu_item "Admin Dashboard", admin_dashboard_path if current_user && current_user.admin? %>
     <%= menu_item "All Borrowers", admin_tenants_path if current_user && current_user.admin? %>
+  <% end %>
+  <%= menu_group pull: :right do %>
```
Comment (EmilyMB) [inline app/views/layouts/_nav_bar.html.erb]: :+1:

File: test/integration/authenticated_lender_test.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    ApplicationController.any_instance.stubs(:current_user).returns(user)
+    visit "/#{project.tenant.slug}"
+
+    within(".row") do
+      click_link_or_button("Lend")
+    end
+    visit root_path
+
+    assert page.has_content?("Help someone today and")
+    assert page.has_content?("complete your loan")
```
Comment (EmilyMB) [inline test/integration/authenticated_lender_test.rb]: Asserting the link might be even better here!

File: test/integration/pending_loans_integration_test.rb [hunk 6]
```diff
@@ -3,16 +3,6 @@
 class PendingLoansIntegrationTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
 
-  test "a cart starts empty" do
```
Comment (EmilyMB) [inline test/integration/pending_loans_integration_test.rb]: Is there a reason to remove this :question:

File: app/views/layouts/_nav_bar.html.erb (hunk 3 above)
Comment (oorion) [inline app/views/layouts/_nav_bar.html.erb]: I can add that

File: test/integration/pending_loans_integration_test.rb (hunk 6 above)
Comment (oorion) [inline test/integration/pending_loans_integration_test.rb]: There's no button for them to click to view their cart.  We could test if the session[:pending_loan] is empty, but that doesn't seem like a good test.

Comment (EmilyMB) [inline test/integration/pending_loans_integration_test.rb]: Ah, right. We removed the pending loan button.

**Comment by EmilyMB** [comment]
Super! It's great to see the site shaping up :+1:

**Pull request merged**